File: Java/179.java
class Solution {
    public String largestNumber(int[] nums) {
        int num,idx;
        String word = "",temp = "";
        for(int i = 0; i < nums.length; i++){
            temp = String.valueOf(nums[i]);
            idx = i;
            for(int j = i + 1; j < nums.length; j++){
                if((String.valueOf(nums[j]) + temp).compareTo(temp + String.valueOf(nums[j])) > 0){
                    temp = String.valueOf(nums[j]);
                    idx = j;
                }
            }
            word += temp;
            num = nums[i];
            nums[i] = nums[idx];
            nums[idx] = num;
        }
        if(word.charAt(0) == '0') return "0";
        return word;
    }
}

File: Python/179.py
class Solution(object):
    def largestNumber(self, nums):
        n = len(nums)
        max_ = ""
        nums = list(map(str,nums))
        for i in range(n):
            idx = 0
            val = nums[0]
            for j in range(1,len(nums)):
                if nums[j] + val > val + nums[j]:
                    val = nums[j]
            nums.remove(val)
            max_ += val
        return str(int(max_))